src/Zipper.elm

```diff
@@ -20,41 +24,84 @@ type alias Zipper a =
     }
 
 
-makeZipper : List a -> a -> List a -> Maybe (Zipper a)
-makeZipper =
-    Debug.todo "make Zipper"
+makeZipper : List a -> a -> List a -> Zipper a
+makeZipper previous current next =
+    { previous = previous, current = current, next = next }
+
+
+moveBackwards : Zipper a -> Maybe (Zipper a)
+moveBackwards zipper =
+    case init zipper.previous of
+        Nothing ->
+            Nothing
+
+        Just choppedPrevious ->
+            Just <|
+                { previous = choppedPrevious
+                , current = withDefault zipper.current <| last zipper.previous
+                , next = zipper.current :: zipper.next
+                }
 
 
 moveForward : Zipper a -> Maybe (Zipper a)
-moveForward =
-    Debug.todo "make moveForward"
+moveForward zipper =
+    map flipZipp << moveForward << flipZipp <| zipper
 
 
-moveBackwards : Zipper a -> Maybe (Zipper a)
-moveBackwards =
-    Debug.todo "make moveBackwards"
+flipZipp : Zipper a -> Zipper a
```

Comment:
Don't abbreviate. `filpZipper` is much better than `flipZipp`. However, you're inside a `Zipper` module, so just `flip` is enough.